index.html

```diff
@@ -20,10 +20,139 @@
                 <h1>TartanHacks</h1>
                 <p>TartanHacks is an annual, student-run hackathon run by <a href="http://www.scottylabs.org/">Scottylabs</a>.</p>
```

Comment:
Use different word choice so that we don't have "student-run" followed by "run by".